### PR TITLE
Add missing none fields

### PR DESCRIPTION
**Pull request by DenDen039**
##### SUMMARY
Add missing none fields to fusion_info module
Some fields were skipped in dictionaries, now they are presented with None values
Fixed fusion_info tests

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
fusion_info

### CODE REVIEW

**Review by sdodsley — DISMISSED** [review]

File: plugins/modules/fusion_info.py [hunk 1]
```diff
@@ -814,6 +814,7 @@ def generate_storserv_dict(module, fusion):
             "display_name": service.display_name,
         }
         # can be None if we don't have permission to see this
+        ss_dict[service.name]["hardware_types"] = None
```
Comment (HonzaHrazdira) [inline plugins/modules/fusion_info.py:817]: Why don't you init this above? 
```Python
        ss_dict[service.name] = {
            "display_name": service.display_name,
            "hardware_types": None
        }
```

File: plugins/modules/fusion_info.py [hunk 2]
```diff
@@ -980,9 +982,12 @@ def generate_volumes_dict(module, fusion):
                     "serial_number": volume.serial_number,
                     "target": {},
                 }
+
+                volume_info[vol_name]["array"] = None
```
Comment (HonzaHrazdira) [inline plugins/modules/fusion_info.py:986]: Why don't you init "array" when the dict is created two lines above?

**Review by HonzaHrazdira — APPROVED** [review]
LGTM. Thanks for the changes.

File: plugins/modules/fusion_info.py [hunk 3]
```diff
@@ -979,10 +981,13 @@ def generate_volumes_dict(module, fusion):
                     "storage_class": volume.storage_class.name,
                     "serial_number": volume.serial_number,
                     "target": {},
+                    "array": None,
                 }
+
                 # can be None if we don't have permission to see this
                 if volume.array is not None:
                     volume_info[vol_name]["array"] = volume.array.name
+
```
Comment (andreepyro) [inline plugins/modules/fusion_info.py:990]: Let's be consistent with the rest of the function (to improve readability of the code).
```suggestion
                    "array": getattr(volume.array, "name", None),
                }
```

**Review by andreepyro — APPROVED** [review]
LGTM

**Pull request merged**